### PR TITLE
Adding additional 10 Accelerate Framework APIs

### PR DESCRIPTION
**Pull request by afaruqui**
Added 8 vDSP APIs and 2 vImage APIs.
Added sample for Accelerate APIs in WOCCatalog.

### CODE REVIEW

File: samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/WOCCatalog.vcxproj [hunk 1]
```diff
@@ -1,241 +1,245 @@
...
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
+    <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
+    <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformMinVersion>10.0.10240.0</WindowsTargetPlatformMinVersion>
+    <ProjectGuid>{7ACE16A9-EA45-422A-80C9-0783F0ED4E9C}</ProjectGuid>
+    <WINOBJC_SDK_ROOT>E:\WinObjC</WINOBJC_SDK_ROOT>
```
Comment (rajsesh-msft) [inline samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/WOCCatalog.vcxproj]: Please change this back to <WINOBJC_SDK_ROOT>..\..\..\..</WINOBJC_SDK_ROOT>

Comment (rajsesh-msft) [inline samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/WOCCatalog.vcxproj]: <WINOBJC_SDK_ROOT>..\..\..\..</WINOBJC_SDK_ROOT>

File: samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/WOCCatalog.vcxproj.filters [hunk 2]
```diff
@@ -1,214 +1,200 @@
...
-      <Filter>Xcode Variable Files</Filter>
-    </Text>
-    <Text Include="WOCCatalog-Release-xcvars.txt">
-      <Filter>Xcode Variable Files</Filter>
-    </Text>
-    <Image Include="..\..\WOCCatalog\Images\tf_bgimage.jpg">
-      <Filter>Images</Filter>
-    </Image>
-  </ItemGroup>
-  <ItemGroup>
```
Comment (rajsesh-msft) [inline samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/WOCCatalog.vcxproj.filters]: package.appxmanifest and WOCCatalog_tempraryKey.pfx got removed.

File: samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/Package.appxmanifest [hunk 3]
```diff
@@ -7,11 +7,11 @@
   IgnorableNamespaces="uap mp">
 
   <Identity
-    Name="141affcc-db48-49b8-befe-db45425604ef"
```
Comment (rajsesh-msft) [inline samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/Package.appxmanifest]: Please revert changes to this file.

**Comment by rajsesh-msft** [comment]
Added some additional code review feedback.  Also please revert the app asset files (all the png files in the samples directory).

**Comment by afaruqui** [comment]
Made changes for the suggested feedback.

**Comment by rajsesh-msft** [comment]
Thank you for your contribution, this change is now merged.

**Pull request merged**